src/plugins/score-plugin-avnd/Crousti/Executor.hpp

```diff
@@ -79,16 +81,21 @@ static midifile_handle loadMidifile(Process::ControlInlet* inlet)
   // Initialize the control with the current soundfile
   if(auto str = filenameFromPort(inlet->value()); !str.isEmpty())
   {
+    std::cerr << "Loading MIDI file" << std::endl;
+
     QFile f(str);
-    if(!f.open(QIODevice::ReadOnly))
-      return {};
+    if(!f.open(QIODevice::ReadOnly)){
+        std::cerr << "File read error for filename " << *str << std::endl;
```

Comment:
In score's code (everything inside the "score/src" folder basically) we use Qt's qDebug() for logging as this way it also shows up in score's console.

Here it'd look like:

    qDebug() << "File read error for filename " << str->c_str();

(it can't print std::string directly and it does not need endl for new lines as it's really a line-by-line debug log